### ISSUE TITLE
<lzaprefix>-SecurityResou-SecurityHubEventsLogSecu-xxxx getting invoked crazy

### ISSUE THREAD

**Issue by DonthineniSagar** [awslabs/landing-zone-accelerator-on-aws#399]
**Describe the bug**
Since we updated to 1.5.0 version security hub event processor lambda getting invoked at crazy in security account costing 1000 of dollars. currently we disabled the eventbus rule to stop the bill.

**To Reproduce**
enable the Eentbus rule. Before the update hardly 100 rule invocations in a day after from Dec11th there are almost million invocations.
<img width="1098" alt="image" src="https://github.com/awslabs/landing-zone-accelerator-on-aws/assets/108803545/13a5c54d-0a63-4c0f-ab2b-f4148fa773da">

**Expected behavior**
Should not invocate the rule and lambda that many times.

**Please complete the following information about the solution:**

- [ ] Version: [e.g. v1.5.1]
1.5.2
> To get the version of the solution, you can look at the description of the created AWS CloudFormation stack used to install the LZA (AWSAccelerator-InstallerStack). For example, "__(SO0199) Landing Zone Accelerator on AWS. **Version 1.5.1.**__". If the description does not contain the version information, you can look at the Parameters of the stack for the **RepositoryBranchName** as that should contain the version number.

- [ ] Region: ap-southeast-2
- [ ] Was the solution modified from the version published on this repository? No
- [ ] If the answer to the previous question was yes, are the changes available on GitHub? NA
- [ ] Have you checked your [service quotas](https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html) for the services this solution uses? NA
- [ ] Were there any errors in the CloudWatch Logs? NA

**Screenshots**
If applicable, add screenshots to help explain your problem (please **DO NOT include sensitive information**).
<img width="1098" alt="image" src="https://github.com/awslabs/landing-zone-accelerator-on-aws/assets/108803545/1b780fbc-3ac3-4658-96b7-761dee6ecaa9">

**Additional context**

**Comment by padebnat** [comment]
Hello @DonthineniSagar,

Thank you for reaching out to the Landing Zone Accelerator team!

Based on the screenshot, it appears that AWS SecurityHub has more findings, which is causing the Lambda to be invoked. Could you please check whether SecurityHub has been modified.

**Comment by DonthineniSagar** [comment]
yes, but seems so weird that how on a partuclar day findings increased by 1000 times, we did not do any changes as per config and cloudtrail. the only change is we deployed LZA 1.5.0 3 days before. We raised a call with AWS support as well. i think lets take it from there!!

**Comment by erwaxler** [comment]
Hi @DonthineniSagar , this has been resolved in our [v1.7.0 release](https://github.com/awslabs/landing-zone-accelerator-on-aws/releases/tag/v1.7.0) by sending events directly to CloudWatch logs without the use of AWS Lambda, I will be closing this issue. Thanks for supporting the Landing Zone Accelerator!
